source/particle/world.cc

```diff
@@ -85,19 +85,26 @@ namespace aspect
     std::string
     World<dim>::generate_output() const
     {
+      std::vector<std::string> filenames;
```

Comment:
Sorry, this was a leftover fragment of code. I originally intended to return a vector of file names but later realized that there wasn;t a clean way of writing these to the statistics file.